stack/keycloak/migration/keycloak-changelog.yml

```diff
@@ -37,4 +37,5 @@ includes:
   - path: add-authorities-basisdaten-wahltage.yml
   - path: add-authorities-basisdaten-ungueltigewahlscheine.yml
   - path: add-authorities-eai-wahlbeteiligung.yml
-  - path: add-authorities-basisdaten-referendumvorlagen.yml
\ No newline at end of file
+  - path: add-authorities-basisdaten-referendumvorlagen.yml
+  - path: add-authorities-basisdaten-wahlen.yml
```

Comment:
```suggestion
  - path: add-authorities-basisdaten-resetwahlen.yml
```

Reply:
Danke! Ich kann es nicht mehr sagen. Vermute durch die ganzen Konflikte die ich zuvor gelöst habe in diesen Keycloak Files, es gelöscht zu haben .. 
Ich habe aber einen anderen Vorschlag. Da wir im Kontext "Wahlen" sowohl resetWahle als auch get und postWahlen haben (von Gerhard) habe ich aus all den Authorities aus diesem Kontext eine Datei gemacht "...-wahlen" wo sowohl reset als auch get und post und delete drinn sind. Sonst wäre nachträglich eine Bereinigung von Keycloak notwendig.
die Änderung hier: 5c1461665e1d700299e440c06d536c9f34cc7524

Reply:
Und kleine Korrektur db29c17edfe1623385a06e1dd0cf6aa4680d7cc3 :) ich habe es getestet, auch die https mit Security, erfolgreich.